src/command.cc

```diff
@@ -170,45 +170,91 @@ AttrPathMixin::fixupAttrPath()
 /* -------------------------------------------------------------------------- */
 
 argparse::Argument &
-RegistryFileMixin::addRegistryFileArg( argparse::ArgumentParser & parser )
+ManifestFileMixin::addManifestFileOption( argparse::ArgumentParser & parser )
 {
-  return parser.add_argument( "--registry-file" )
-    .help( "The path to the 'registry.json' file." )
-    .required()
+  return parser.add_argument( "--manifest" )
+    .help( "The path to the 'manifest.{toml,yaml,json}' file." )
     .metavar( "PATH" )
     .action( [&]( const std::string & strPath )
-             { this->setRegistryPath( strPath ); } );
+             { this->manifestPath = nix::absPath( strPath ); }
+           );
 }
 
-void
-RegistryFileMixin::setRegistryPath( const std::filesystem::path & path )
+argparse::Argument &
+ManifestFileMixin::addManifestFileArg( argparse::ArgumentParser & parser )
+{
+  return parser.add_argument( "manifest" )
+    .help( "The path to the 'manifest.{toml,yaml,json}' file." )
+    .metavar( "MANIFEST-PATH" )
+    .action( [&]( const std::string & strPath )
+             { this->manifestPath = nix::absPath( strPath ); }
+           );
+}
+
+std::filesystem::path
+ManifestFileMixin::getManifestPath()
 {
-  if ( path.empty() )
+  auto tryFile = [&]( const std::string & path ) -> bool
+  {
+    std::filesystem::path absPath = nix::absPath( path );
+    if ( std::filesystem::exists( absPath ) )
+      {
+        this->manifestPath = absPath;
+        return true;
+      }
+    return false;
+  };
+
+  bool have = this->manifestPath.has_value() ||
+              tryFile( "manifest.toml" ) ||
+              tryFile( "manifest.yaml" ) ||
+              tryFile( "manifest.json" ) ||
+              tryFile( ".flox/manifest.toml" ) ||
+              tryFile( ".flox/manifest.yaml" ) ||
+              tryFile( ".flox/manifest.json" );
```

Comment:
suggestion nonblocking: drop these and make `--manifest` required

Reply:
Future proofeded

Reply:
Y'all can delete the JSON and TOML options after doing user A/B testing and I swear I'll never utter the phrase "I told you so" 😶

Reply:
I went ahead and changed it, and removed anything that tried to read from `PWD`